tests/framework/PostProcessors/Validation/PhysicsGuidedCoverageMapping/.ravenStatus

```diff
@@ -0,0 +1 @@
+Success
```

Comment:
.DS_Store and .ravenStatus should not be added to the repository.

Reply:
> @ShimingYIN Could you resolve all the comments? I will try to review it before our meeting.

@wangcj05 Sorry for the delay. I have removed the class "PhysicsGuidedCoverageMapping_snapshot" and combined it with the class "PhysicsGuidedCoverageMapping". And the SETH data applied for snapshot_PCM has been reduced, thus the test time is around 30sec now. The '.DS_Store' and '.ravenStatus' files have been removed and 'pythonfmu.egg-info' directory is not commited now.

Reply:
Thanks for fixing.